openupgrade_scripts/scripts/hr/17.0.1.1/end-migration.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2024 Viindoo Technology Joint Stock Company (Viindoo)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from openupgradelib import openupgrade
+
+
+def _merge_employee_contact(env):
+    """
+    Performing merge 'address_home_id' into 'work_contact_id'
+    """
+    partner_merge_wizard = env["base.partner.merge.automatic.wizard"]
+    partner = env["res.partner"]
+    env.cr.execute(
+        """
+        SELECT address_home_id, work_contact_id
+        FROM hr_employee
```

Comment:
```suggestion
        FROM hr_employee
        WHERE work_contact_id IS NOT NULL AND address_home_id IS NOT NULL
        AND work_contact_id<>address_home_id
```

as the merge function chokes if any of them is not set or they are the same, which all are perfectly valid settings in v16